vsphere/vsphere.go

```diff
@@ -629,12 +631,19 @@ func (ops *vsphereOps) Expand(
 
 	task, err := vm.Reconfigure(ctx, spec)
 	if err != nil {
+		if strings.Contains(err.Error(), vmdkNotFoundErrorMsg) {
+			return 0, fmt.Errorf(err.Error() + svmotionErrorMsg)
+		}
 		return 0, err
 	}
 
 	err = task.Wait(ctx)
 	if err != nil {
-		return 0, fmt.Errorf("error resizing vmdk: %s due to:  %s", vmdkPath, err)
+		errMsg := fmt.Errorf("error resizing vmdk: %s due to:  %s", vmdkPath, err)
```

Comment:
- print this errMsg here
- errMsg = fmt.Errorf("error resizing vmdk: %s. Path not found. Retry pool expansion, if a storage vMotion operation was in progress during expansion.", vmdkPath)
- Declare the error string as a global variable and use in line 635 as well.